Wizard/Wizard.js

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
```

Comment:
@ShadyXV  this PR as the name suggests is about updating prop type for nav component,  please remove newly added files